docs/products/clickhouse/concepts/data-integration-overview.md

```diff
@@ -107,6 +109,23 @@ Depending on a data source type, Aiven for ClickHouse supports different integra
 | ClickHouse       | <ConsoleIcon name="tick"/> | <ConsoleIcon name="cross"/>  | <ConsoleIcon name="tick"/>     |
 | Amazon S3        | <ConsoleIcon name="tick"/> | <ConsoleIcon name="cross"/>  | <ConsoleIcon name="tick"/>      |
 
+## Data flow and residency in integrations
+
+### How data is pulled in and distributed
+
+If you integrate a multi-node Aiven for ClickHouse service with another Aiven-managed
+service or an external endpoint, one of the Aiven for ClickHouse service nodes connects
+to the other integrating entity. If the data that is read from this entity is persisted in
+the Aiven for ClickHouse service, this data is replicated across all the nodes in the
+cluster.
+
+### Where data resides upon integration
+
+When you integrate Aiven for ClickHouse with another Aiven-managed service or an external
+endpoint, by default data lives outside the Aiven for ClickHouse service, in its original
+source. Queries aggregate data across sources, and the integrated data is accessed as
```

Comment:
```suggestion
endpoint, by default, data resides outside the Aiven for ClickHouse service in its original 
source. Queries aggregate data across sources, and the integrated data is accessed as
```
I feel that 'data lives' is informal and uses conversational language. Perhaps 'data resides' or 'data is stored'.